app/models/story.rb

```diff
@@ -326,12 +326,12 @@ def update_published_to_released
 
   def set_status
     av_errors = ''
-    if audio_versions.empty?
-      av_errors << "Story '#{title}' has no audio."
+    if audio_files.empty?
```

Comment:
After trying some different things, I realized the Story already checks for `audio_versions.empty?`.  From there, it's not very controversial to test for `audio_files.empty?`.

This has the added bonus of not breaking Publish.  Setting an `AudioVersion.status_message` would show up right away in the Publish UI (in the audio-uploader widget).  Whereas a `Story.status_message` only shows up if Publish can't find anything else wrong and you're about to publish a story.

Reply:
+1 So basically this is saying, no matter what configuration of audio versions, we expect at least one audio file in a publishable story

Reply:
Exactly.  It seems like this was already a business requirement, and enforced with _most_ audio versions (those with a "duration > 0" validation).  But we just weren't catching this edge case.